examples/tutorials/05_ordinary_models.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p>That's not a very good model since it contains 2 extremely large dense layers.</p><p>Could you remove the last <code>Linear</code> from <code>resnet50</code> and use global pooling  in the <code>head</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>